app/ide-desktop/.gitignore

```diff
@@ -1,3 +1,3 @@
 node_modules
-dist
+**/dist
```

Comment:
I believe that previous gitignore was ignoring "dist" in any sub-directory. I think this change is not needed.

Reply:
true, will revert!